helper/schema/schema.go

```diff
@@ -1176,6 +1183,16 @@ func (m schemaMap) validateType(
 			"%q: [REMOVED] %s", k, schema.Removed))
 	}
 
+	if schema.ValidateFunc != nil {
```

Comment:
Diving in - I realized that I was trying to do this by putting the call late in the Validate process, but I wasn't thinking about the fact that the error list passes through instead of early-returning. Fixed up now!